tests/generateCSV.test.js

```diff
@@ -0,0 +1,19 @@
+const { getRandomTenPhotoReferences, createEntry, writeData } = require('../database/generateCSV.js');
+
+test('getRandomTenPhotosReferences should generate an array of 10', () => {
+  expect(getRandomTenPhotoReferences().length).toBe(10);
+});
+
+test('getRandomTenPhotosReferences should generate random arrays', () => {
+  const array1 = getRandomTenPhotoReferences();
+  const array2 = getRandomTenPhotoReferences();
+  expect(array1).not.toBe(array2);
+});
+
+test('createEntry should return a string', () => {
+  expect(typeof createEntry(1)).toBe('string');
+});
+
+test('createEntry should return a string with the passed in value as the first character', () => {
+  expect(createEntry(1)[0]).toBe('1');
+});
```

Comment:
If you could include a test to check the shape of the generated string, that would be very helpful test, since it will ensure correct seeding

Reply:
Thanks Yogita! I added two more tests!